packages/lib-utils/src/components/status/StatusBox.tsx

```diff
@@ -0,0 +1,72 @@
+// @ts-nocheck
+import * as React from 'react';
+import { Alert, Bullseye, Spinner, Text, Title } from '@patternfly/react-core';
+import * as _ from 'lodash';
+
+type AccessDeniedProps = {
+  message?: string;
+};
+
+export const AccessDenied: React.FC<AccessDeniedProps> = ({ message }) => {
+  return (
+    <Bullseye>
+      <Title headingLevel="h2">Restricted Access</Title>
+      <Text>You don&apos;t have access to this section due to cluster policy.</Text>
+      <Alert isInline variant="danger" title={'Error details'}>
+        {message}
+      </Alert>
+    </Bullseye>
+  );
+};
+
+type StatusBoxProps = {
+  label?: string;
+  loadError?: any;
+  loaded?: boolean;
+  data?: any;
+  unfilteredData?: any;
+  // skeleton?: React.ReactNode;
+  NoDataEmptyMsg?: React.ComponentType;
+  EmptyMsg?: React.ComponentType;
+  children?: React.ReactElement;
+};
+
+export const StatusBox: React.FC<StatusBoxProps> = (props) => {
+  const { data, EmptyMsg, loadError, loaded, children, NoDataEmptyMsg, unfilteredData } = props;
+
+  if (loadError) {
+    const status = _.get(loadError, 'response.status');
+    if (status === 404) {
```

Comment:
we could probably do a `switch` statement on the `status` so it's easier to add/catch new cases in the future.